clients/src/main/java/org/apache/kafka/common/protocol/types/Struct.java

```diff
@@ -99,6 +99,10 @@ public String get(Field.NullableStr field) {
         return getString(field.name);
     }
 
+    public Boolean get(Field.Boolean field) {
```

Comment:
Should the result type be `boolean`? @hachikuji, was it intentional that the result type of various `get` methods is returning the boxed type?

Reply:
The value is already boxed, but I can't think of a great reason why we should return the boxed type. Other than strings, the primitive types don't allow null.

Reply:
That's what I was thinking, it's more obvious that null is not expected if it's a primitive.

Reply:
@ijuma @hachikuji Ok, I changed it to `boolean`. However, The other methods seem to be using Long, Integer, Short etc. as well. Do we want to change them with this PR as well? Or raise a JIRA for it?

Reply:
Let's just do it in this PR. Thanks!

Reply:
Ok, I changed it also for Long, Short, Integer and Byte. It should be fixed now.